### ISSUE TITLE
osquery 5.0 breaking changes

### ISSUE THREAD

**Issue by directionless** [autopkg/keeleysam-recipes#138]
Hi! You probably need to update your autopkg in light of the osquery 5.0 breaking changes. 
* macOS packages are now code signed by the osquery foundation
* If you use the default package, it’s now in /opt/osquery
* There’s an app bundle. this was needed to support the endpoint security permissions

See https://github.com/osquery/osquery/releases/tag/5.0.1 for more (edited)

**Comment by mpanighetti** [comment]
Should be fixed with #146.

**Comment by arubdesu** [comment]
In the meantime people are using [jbaker10-recipes](https://github.com/autopkg/jbaker10-recipes/tree/master/osquery)

**Comment by mpanighetti** [comment]
Yep, I just sent the same PR to both repos and that one merged first. :D
